cpp/arcticdb/version/symbol_list.hpp

```diff
@@ -30,7 +30,8 @@
 #include <mutex>
 
 // FUTURE(GitHub #297)
-#define SYMBOL_LIST_RUNTIME_LOG(...) ARCTICDB_RUNTIME_DEBUG(log::version(), "Symbol List: {}: ", __func__, __VA_ARGS__)
+#define SYMBOL_LIST_RUNTIME_LOG(message) ARCTICDB_RUNTIME_DEBUG(log::version(), "Symbol List: {}: " message, __func__)
```

Comment:
We are jumping through a lot of hoops here to avoid creating a new log channel, which would not only indicate that the logging relates to the symbol list but would allow us to toggle it on and off independently

Reply:
That's fair - I just didn't want to change it in this set of changes really.